libs/sof/epipolar_curves.cpp

```
@@ -0,0 +1,163 @@
+constexpr float kAutoBaselineSmall = 0.075f;  // ~7.5 cm — small stereo (5–10 cm range midpoint)
+constexpr float kAutoBaselineLarge = 0.5f;    // 0.5 m — KITTI-scale
+constexpr float kAutoMinDepthSmall = 0.1f;    // 10 cm
+constexpr float kAutoMinDepthLarge = 7.f;     // 7 m
+constexpr float kAutoMaxDepthSmall = 20.f;    // 20 m
+constexpr float kAutoMaxDepthLarge = 1000.f;  // 1 km
+
+// Fills any non-positive `min_depth` / `max_depth` with baseline-interpolated defaults, then
+// throws if the resulting range is not strictly positive and monotonic. Non-positive is the
+// "auto" trigger (covers -1 sentinels and accidental 0s that would otherwise reach `std::log`).
+void AutoDetectDepthRange(const float baseline, float& min_depth, float& max_depth) {
+  const float t = std::clamp((baseline - kAutoBaselineSmall) / (kAutoBaselineLarge - kAutoBaselineSmall), 0.f, 1.f);
```

Clang (14.0.6)

[warning] variable name 't' is too short, expected at least 3 characters

(readability-identifier-length)